dali/dfu/dfuwu.cpp

```diff
@@ -2077,6 +2077,16 @@ class CDFUoptions: public CLinkedDFUWUchild, implements IDFUoptions
         queryRoot()->setProp("@umask",val);
 
     }
+    int getExpireDays() const
+    {
+        if (queryRoot()->hasProp("@expireDays"))
```

Comment:
minor: No need to check this condition - since it will return -1 if the attribute doesn't exist.